### PR TITLE
Add com.studio08.xbplay

### PR DESCRIPTION
**Pull request by alexwarddev1230**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.studio08.xbplay.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+app-id: com.studio08.xbplay
```
Comment (hfiguiere) [inline com.studio08.xbplay.yml:1]: `studio08.com` doesn't seem to be related, not do I see it mentionned anywhere. Is it really controlled by the upstream developers?

File: com.studio08.xbplay.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id type="desktop">com.studio08.xbplay</id>
+    <launchable type="desktop-id">com.studio08.xbplay.desktop</launchable>
+    <name>xbPlay: Remote Play for Xbox</name>
+    <summary>Remote play from your console or xCloud</summary>
+    <metadata_license>CC-BY-SA-4.0</metadata_license>
```
Comment (hfiguiere) [inline com.studio08.xbplay.metainfo.xml:7]: `CC0-1.0` is prefered for the meta data.

File: com.studio08.xbplay.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+      # add desktop and metainfo files
+      - install -Dm644 com.studio08.xbplay.desktop -t /app/share/applications/
+      - install -Dm644 com.studio08.xbplay.metainfo.xml -t /app/share/metainfo/
+
+      # install the app runner
+      - install -Dm755 -t /app/bin/ run.sh
+    sources:
+      - type: archive
+        url: https://www.dropbox.com/s/333ffftl6qimhdq/data_1.1.0.zip?dl=1
```
Comment (hfiguiere) [inline com.studio08.xbplay.yml:43]: you download a binary but still build an aarch64 AND x86_64. One of them is unlikely to work.

File: com.studio08.xbplay.yml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
```
Comment (hfiguiere) [inline com.studio08.xbplay.yml:13]: I guess you don't need GL?

File: com.studio08.xbplay.desktop [hunk 5]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=xbPlay: Remote Play for Xbox
+Comment=Remote play from your console or xCloud.
+Icon=com.studio08.xbplay.png
```
Comment (hfiguiere) [inline com.studio08.xbplay.desktop:5]: Icon is NEVER with an extension.

File: com.studio08.xbplay.yml (hunk 3 above)
Comment (alexwarddev1230) [reply com.studio08.xbplay.yml:43]: Thanks, realizing now that I only support x86_64 at this point. How do I prevent arm aarch64 builds?

File: com.studio08.xbplay.yml (hunk 1 above)
Comment (alexwarddev1230) [reply com.studio08.xbplay.yml:1]: I am the only developer of this app, there are no upstream developers?

File: com.studio08.xbplay.yml (hunk 4 above)
Comment (alexwarddev1230) [reply com.studio08.xbplay.yml:13]: I don't think so? Its working fine without that. I simply copied what the example electron app had (as this is an electron app)

File: com.studio08.xbplay.yml (hunk 3 above)
Comment (alexwarddev1230) [reply com.studio08.xbplay.yml:43]: Guessing something like this: https://github.com/flathub/chat.rocket.RocketChat/blob/master/flathub.json will do it?

File: com.studio08.xbplay.desktop (hunk 5 above)
Comment (alexwarddev1230) [reply com.studio08.xbplay.desktop]: Thanks, resolved.

File: com.studio08.xbplay.yml (hunk 3 above)
Comment (alexwarddev1230) [reply com.studio08.xbplay.yml:43]: Added a file like that ^. Let me know if there is anything else missing there.

File: com.studio08.xbplay.yml (hunk 1 above)
Comment (alexwarddev1230) [reply com.studio08.xbplay.yml:1]: @hfiguiere  dang, I have `.net` not `.com`. Looks like I'll need to change literally everything? :(

**Comment by alexwarddev1230** [comment]
Since I have to change the name I am closing out this PR and opening a new one on a different branch.

**Comment by hfiguiere** [comment]
the branch name doesn't matter. just the file names.
